packages/react/src/ui-react/components/form/checkbox.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { FormControlProps } from "./form-control";
+
+import { Checkbox as FluentCheckbox } from "@fluentui/react/lib/Checkbox";
+
+export interface CheckboxProps<V>
+    extends FormControlProps<boolean | undefined> {
+    checked?: boolean;
+    defaultChecked?: boolean;
+    indeterminate?: boolean;
+    defaultIndeterminate?: boolean;
+    boxSide?: "start" | "end";
+}
+
+export function Checkbox<V>(props: CheckboxProps<V>): JSX.Element {
+    if (props.hidden) {
+        return <></>;
+    }
+
+    return (
+        <FluentCheckbox
+            id={props.id}
```

Comment:
Just a minor note that instead of passing individual props, you can pass the entire props object with the spread operator, e.g., `<FluentCheckbox ...props onChange={/* ... */} />`.

Reply:
Thanks! Just changed that for both form controls